pyproject.toml

```diff
@@ -37,7 +37,7 @@ schematic = "schematic.__main__:main"
 
 [tool.poetry.dependencies]
 python = ">=3.9.0,<3.11"
-
+sphinx = "7.3.7"
```

Comment:
Question for all - should these be part of the dev dependencies?

Reply:
cc @GiaJordan @andrewelamb

Reply:
Yes that makes sense since one installing/using this as a CLI would not need sphinx

Reply:
The original `README` instructions assumed only developers/contributors would be using `poetry`, so I think that's why the sphinx libraries were added to the main dependencies. I'll create a separate group for documentation-related librarie, move everything there, and update the `README` accordingly.